Restructure some pages to create new "Connecting" section (#633)
localstack/localstack-docs: src/content/docs/aws/connecting/index.mdx

---
title: Overview

> description: Connect to LocalStack using the same AWS tools you already use — CLI, SDKs, IaC, the web console, and IDE integrations.

Reviewer: ```suggestion
description: Connect to LocalStack using your favorite AWS tools, such as CLI, SDKs, IaC, the web console, and IDE integrations.
```

template: doc
editUrl: false
sidebar:
    order: 1
---

import SectionCards from '../../../../components/SectionCards.astro';

> If you already run an application on AWS, you almost certainly connect to it through familiar tools — the AWS CLI, an AWS SDK, an Infrastructure as Code tool such as Terraform or AWS CDK, or an IDE plugin. Running that same application against LocalStack means continuing to use those tools; you only need to point them at LocalStack instead of AWS.

Reviewer: ```suggestion
If you already run an application on AWS, you almost certainly connect to it through familiar tools, such as the AWS CLI, an AWS SDK, an Infrastructure as Code tool such as Terraform or AWS CDK, or an IDE plugin. Running that same application against LocalStack means continuing to use those tools; you only need to point them at LocalStack instead of AWS.
```
